spec/unit/lib/temporal/workflow/context_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'temporal/workflow'
+require 'temporal/workflow/context'
+
+class MyTestWorkflow < Temporal::Workflow; end
+
+describe Temporal::Workflow::Context do
+  let(:state_manager) { instance_double('Temporal::Workflow::StateManager') }
+  let(:dispatcher) { instance_double('Temporal::Workflow::Dispatcher') }
+  let(:metadata) { instance_double('Temporal::Metadata::Workflow') }
+  let(:workflow_context) {
+    Temporal::Workflow::Context.new(state_manager, dispatcher, MyTestWorkflow, metadata, Temporal.configuration)
+  }
+
+  describe '#upsert_search_attributes' do
```

Comment:
to make it useful we also need a spec that ensures a correct command is added when this method gets called